pkg/network/ovn_kubernetes.go

```diff
@@ -174,12 +174,8 @@ func boostrapOVN(kubeClient client.Client) (*bootstrap.BootstrapResult, error) {
 		return nil, err
 	}
 
-	if len(masterNodeList.Items) == 0 {
-		return nil, fmt.Errorf("unable to bootstrap OVN, no master nodes found")
-	}
-
 	if len(masterNodeList.Items) < 3 {
```

Comment:
Yeah, this should not have been merged. Could you remove this check altogether? Nothing says that we should ever have X master nodes.

Reply:
Edit: just saw that you did. Sorry about that

Reply:
yeah, I realized afterward, this is an attempt to deal with the problem #421 is trying to fix